bitcoin/src/amount.rs

```diff
@@ -33,6 +34,8 @@ use crate::prelude::*;
 pub enum Denomination {
```

Comment:
Looks like this should be `#[non_exhaustive]`?

Reply:
yeah it should be non-exhastive unless there will be no other additions.  That's probably out of scope for this PR though.

Reply:
I wouldn't mind either. It does have three ACKs but I don't want to prolong the release by adding this unless you need it ASAP.

Reply:
I can just use the PR as the source for now until it's merged so it's not a rush.